source/pigeonhole-man/sieve-filter.1.rst.in

```diff
@@ -0,0 +1,266 @@
+in a Sieve script caused many messages to be delivered incorrectly.
+Using the sieve-filter tool it is possible to apply a Sieve script on
+all messages in a particular *source-mailbox*, making it possible to
+delete messages, to store them in a different mailbox, to change their
+content, and to change the assigned IMAP flags and keywords. Attempts to
+send messages to the outside world are ignored by default for obvious
+reasons, but, using the proper command line options, it is possible to
+capture and handle outgoing mail as well.
+
+If no options are specified, the sieve-filter command runs in a
+simulation mode in which it only prints what would be performed, without
+actually doing anything. Use the **-e** option to activate true script
+execution. Also, the *source-mailbox* is opened read-only by default,
+meaning that it normally always remains unchanged. Use the **-W** option
+to allow changes in the *source-mailbox*.
+
+Even with the **-W** option enabled, messages in the *source-mailbox*
+are only potentially modified or moved to a different folder. Messages
+are never lost unless a *discard-action* argument other than **keep**
+(the default) is specified. If the Sieve filter decides to store the
+message in the *source-mailbox*, where it obviously already exists, it
+is never duplicated there. In that case, the IMAP flags of the original
+message can be modified by the Sieve interpreter using the *imap4flags*
+extension, provided that **-W** is specified. If the message itself is
+modified by the Sieve interpreter (e.g. using the *editheader*
+extension), a new message is stored and the old one is expunged.
+However, if **-W** is omitted, the original message is left untouched
+and the modifications are discarded.
+
+CAUTION
+-------
+
+Although this is a very useful tool, it can also be very destructive
+when used improperly. A small bug in your Sieve script in combination
+with the wrong command line options could cause it to discard the wrong
+e-mails. And, even if the *source-mailbox* is opened in read-only mode
+to prevent such mishaps, it can still litter other mailboxes with
+spurious copies of your e-mails if your Sieve script decides to do so.
+Therefore, users are advised to read this manual carefully and to use
+the simulation mode first to check what the script will do. And, of
+course:
+
+**MAKING A BACKUP IS IMPERATIVE FOR ANY IMPORTANT MAIL!**
+
+OPTIONS
+=======
+
+**-c**\ *config-file*
+   Alternative Dovecot configuration file path.
+
+**-C**
+   Force compilation. By default, the compiled binary is stored on disk.
+   When this binary is found during the next execution of
+   **sieve-filter** and its modification time is more recent than the
+   script file, it is used and the script is not compiled again. This
+   option forces the script to be compiled, thus ignoring any present
+   binary. Refer to **sievec**\ (1) for more information about Sieve
+   compilation.
+
+.. include:: option-d.inc
```

Comment:
The description for -d included here is for sievec and not for the -d <dumpfile> structure used by other commands like sieve-filter. This problem is therefore also present for sieve-test.